### ISSUE TITLE
Calculated Sun Position and events as entities in HO

### ISSUE THREAD

**Issue by mysleinic** [rejuvenate/lovelace-horizon-card#108]
Hi

Is there any chance that the calculated data inside the card could be exposed as entities in HA and later use to run automations?

In my particular case - I would love to set my aquarium lighting to match the same weather and sun status as chosen tropical location.
I've done deep dive into the topic, elevation and azimuth, formulas, horizon typescript code... I found some sketchy solution to my problem... but what a convenient it would be to have it just available at hand ;)

It might be useful for plenty of aquarist, especially those willing to have biotope aquariums.

Best regards,
Artur Pyka

**Comment by CSJudd** [comment]
YES PLEASE :). This exposure of the data would be a God Send for many things I have in mind :) Let me know if there is a way I can help

**Comment by philpownall** [comment]
I support this proposal - I would very much like to make my 3D Moon tracker run without Astroweather, and maybe migrate it to a platform in ESPHome as well so that it can run without being attached to HA.

![Moon Tracker](https://github.com/rejuvenate/lovelace-horizon-card/assets/24682344/7db0702b-cab1-4a6b-8908-c2b0b2d9866e)
![SunMoonTracker](https://github.com/rejuvenate/lovelace-horizon-card/assets/24682344/f5a7a482-ab9e-4f2e-b325-167076b1c827)

**Comment by ThomDietrich** [comment]
Consolidating to avoid fragmentation — this looks like the same topic as #164, which we're keeping as the tracker. Please follow along and add details there. (Closing as duplicate; reopen if you think it's distinct.)
